### DISCUSSION TITLE
Broken image .eps -> .jpg

### DISCUSSION THREAD

**Discussion by grundkvist** [ImageMagick/ImageMagick#2793] [Help] [answered]
When I run "magick 1.eps 1.jpg" on my MacBook I get two output files. One is ok, the other one is broken. When we run the same string on our linux server we seem to only get the broken file. Why?
![1-0](https://user-images.githubusercontent.com/12462044/98081842-1c060f00-1e78-11eb-9d10-4f9ed46231c5.jpg)
![1-1](https://user-images.githubusercontent.com/12462044/98081848-1e686900-1e78-11eb-8312-8efa49921119.jpg)

**Comment by t246246** [comment, marked as answer]
As you do not attach your input eps file, it is just a guess, but your eps is not a standard eps, but proprietary format. See https://github.com/ImageMagick/ImageMagick/discussions/2661
Some older version can recognize only main image in it.  (Also this is a  guess, your linux and mac has different version of ImageMagick and/or Ghostscript.)

**Reply by grundkvist** [reply]
[1.eps.zip](https://github.com/ImageMagick/ImageMagick/files/5486269/1.eps.zip)

**Reply by t246246** [reply]
Yes it is EPT.

$ magick identify 1.eps
1.eps[0] EPT 175x415 175x415+0+0 16-bit sRGB 100965B 0.109u 0:11.778
1.eps[1] TIFF 175x415 175x415+0+0 1-bit Grayscale Gray 48982B 0.000u 0:00.002

All I can say now us it use newer version as possible, and select only the first image.

% magick 1.eps[0] 1.jpg

See https://github.com/ImageMagick/ImageMagick/discussions/2661#discussioncomment-89934

**Reply by grundkvist** [reply]
Thanks!
